### PR TITLE
Fix GHA matrix

### PR DESCRIPTION
**Pull request by szepeviktor**
kép

😢 

Only primitive values.

### CODE REVIEW

File: .github/workflows/code_analysis.yaml [hunk 1]
```diff
@@ -25,12 +24,12 @@ jobs:
             # see https://github.com/shivammathur/setup-php
             -   uses: shivammathur/setup-php@v2
                 with:
-                    php-version: 8.1
+                    php-version: "8.1"
```
Comment (szepeviktor) [inline .github/workflows/code_analysis.yaml]: YAML thinks this is a float.

May I quote all YAML strings?

**Comment by matthiasnoback** [comment]
PRs are very welcome, thanks!

**Comment by szepeviktor** [comment]
It is very green https://github.com/szepeviktor/recipes-for-decoupling-phpstan-rules/runs/7168165965?check_suite_focus=true 🍏

**Comment by matthiasnoback** [comment]
I wasn't even aware that this was something in need of fixing :)

**Pull request merged**